multisrc/overrides/wpmangastream/mareceh/src/Mareceh.kt

```diff
@@ -0,0 +1,18 @@
+package eu.kanade.tachiyomi.extension.id.mareceh
+
+import eu.kanade.tachiyomi.lib.ratelimit.RateLimitInterceptor
+import eu.kanade.tachiyomi.multisrc.wpmangastream.WPMangaStream
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+class Mareceh : WPMangaStream("Mareceh", "https://mareceh.com", "id") {
```

Comment:
Potentially the URL paths have changed and the old URLs are unsupported. If it's the case, please also bump the `versionId`.

```suggestion
class Mareceh : WPMangaStream("Mareceh", "https://mareceh.com", "id") {

    override val versionId = 2

```